site/en/docs/web-platform/region-capture/index.md

```diff
@@ -111,7 +111,7 @@ If unsuccessful, the Promise will be rejected. This will happen if:
 The `cropTo()` method is exposed on any tab-capture video track, and not just for self-capture. It is therefore advisable to check if the user selected the current tab, before attempting to crop the track. This can be accomplished using [Capture Handle]. It is also possible to ask the browser to nudge the user towards self-capture using [`preferCurrentTab`].
```

Comment:
See this:
> - The current track is not a self-capture video track; see below.

And this:
> The `cropTo()` method is exposed on any tab-capture video track, and not just for self-capture.

And this:
>It is also possible to ask the browser to nudge the user towards self-capture using [`preferCurrentTab`].

I suggest scanning the doc for "self" and "own".

Reply:
I've tried to update the article so that when talking about the Region Capture API in general, I don't use self-capture and when talking about Chrome specifically, I add mention that it only supports self-capture video tracks.

Reply:
Such a nuance would have eluded me if I were the reader. I think it would be wiser to [KISS](https://en.wikipedia.org/wiki/KISS_principle). How about we wait until we implement this, and then we can overhaul the doc to call out and demonstrate other-tab-cropping, possibly with a useful demo too?